app/models/declaration/Parties.scala

```diff
@@ -16,8 +16,9 @@
 
 package models.declaration
 
+import forms.common.YesNoAnswer
 import forms.declaration.consignor.ConsignorDetails
-import forms.declaration.{CarrierDetails, ConsigneeDetails, DeclarantDetails, DeclarantIsExporter, ExporterDetails, IsExs}
+import forms.declaration.{CarrierDetails, ConsigneeDetails, DeclarantDetails, DeclarantIsExporter, ExporterDetails, PersonPresentingGoodsDetails, _}
```

Comment:
Is it not better to just import it with `_`. You have this at the end, so probably rest of the import's parts are unnecessary